poc_mobile_verifier/src/owner_shares.rs

```diff
@@ -303,7 +303,12 @@ mod test {
             },
         ];
 
-        let heartbeats: Heartbeats = heartbeats.into_iter().collect();
+        let mut heartbeats: Heartbeats = heartbeats.into_iter().collect();
+
+        // Populate with the minimum required number of timestamps
+        for (_, hb) in heartbeats.heartbeats.iter_mut() {
+            hb.timestamps = std::iter::repeat(timestamp).take(12).collect();
```

Comment:
should these timestamps each be from different hours to capture the actual requirement? unless i'm misunderstanding this it's just going to get twelve of the _same_ timestamp

Reply:
Since the logic is entirely in the sql query I would argue no, that would give the impression that we are doing such checks on the client side, which we are not. The only thing that the client does is check that the number of timestamps is greater than 12

Reply:
makes sense; i missed that we were _only_ tracking the hourly differences in the db